plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/bean/BitstampBeanV1.java

```diff
@@ -32,6 +38,13 @@
     private Currency rateCurrency;
     private Currency feeCurrency;
 
+    @Parsed(field = "Type")
+    public void setType(String field) {
+        if(!field.equals("Market")) {
```

Comment:
Change this to !"Market".equals(field). This way Null Pointer Exception will not happen in field is null.